libraries/api-documenter/src/index.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as os from 'os';
+import * as path from 'path';
+
+const myPackageJsonFilename: string = path.resolve(path.join(
+  module.filename, '..', '..', 'package.json')
+);
+const myPackageJson: { version: string } = require(myPackageJsonFilename);
```

Comment:
@iclanton  Yes in general, because require() does not report syntax errors well, and it will execute malicious scripts embedded in the files.

However, IMO it is okay to use require for a static asset that is embedded in your package, because:
- It won't contain any mistakes, because users cannot change it
- It won't contain malicious scripts, because users cannot change it


---
In reply to: [134040687](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134040687) [](ancestors = 134040687)